semantics/executable-spec/src/Control/State/Transition/Extended.hs

```diff
@@ -253,6 +256,15 @@ type family EventReturnType ep sts a :: Type where
   EventReturnType 'EventPolicyReturn sts a = (a, [Event sts])
   EventReturnType _ _ a = a
 
+class EventReturnTypeRep ert where
```

Comment:
maybe `EventReturnTypeRep` -> `ReifyEventPolicy` or something

Reply:
I went for consistency with `RuleTypeRep`, in the same file.